### PR TITLE
New package: CheatBreaker.CheatBreaker version 2.4.2

### PR DESCRIPTION
**Pull request by ypurpl**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136494)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 661.)

File: manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CheatBreaker.CheatBreaker
+PackageVersion: 2.4.2
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2023-10-19
```
Comment (Exorcism0666) [inline manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml:10]:
```suggestion
ReleaseDate: 2023-10-19
InstallerSwitches:
    Silent: /?
    SilentWithProgress: /?
```

**Comment by vvirtues** [comment]
The app doesnt seem to support any known arguments, but it installs with no interaction and a progress bar. How do I get it to work?

File: manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml (hunk 1 above)
Comment (vvirtues) [reply manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml:10]: doesnt support flags, but is the same as `SilentWithProgress`

**Comment by stephengillie** [comment]
> Automatic Validation ended with:
> 
> > 
> 
> (Automated response - build 661.)

image

(The image was opened in the background and I wasn't sure which PR it came from.)

File: manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 2.4.2
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2023-10-19
+InstallerSwitches:
+  SilentWithProgress: 
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml:14]:
```suggestion
- Architecture: x64
```
Our `x86` validations are performed on 32-bit OSes.

**Comment by vvirtues** [comment]
aight bet

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: CheatBreaker.CheatBreaker
+PackageVersion: 2.4.2
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2023-10-19
+InstallerSwitches:
+  SilentWithProgress: 
+Installers:
```
Comment (stephengillie) [inline manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml:13]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Oracle.JavaRuntimeEnvironment
Installers:
```

(Automated response - build 669.)

File: manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CheatBreaker.CheatBreaker
+PackageVersion: 2.4.2
+PackageLocale: en-US
+Publisher: Not affiliated with CheatBreaker, LLC
```
Comment (Trenly) [inline manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.locale.en-US.yaml:7]:
```suggestion
Publisher: Cheatbreaker
```
This should match what the application writes to control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**